templates/registration/password_reset_confirm.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<div class="container">
+
+    <div class="row">
+        <div class="col-md-12">
+
+            {% if validlink %}
+                <p>Please enter (and confirm) your new password.</p>
+                <form action="" method="post">
+                    <div style="display:none">
+                        <input type="hidden" value="{{ csrf_token }}" name="csrfmiddlewaretoken">
+                    </div>
+                    <table>
+                        <tr>
+                            <td>{{ form.new_password1.errors }}
+                                <label for="id_new_password1">New password:</label></td>
+                            <td>{{ form.new_password1 }}</td>
+                        </tr>
+                        <tr>
+                            <td>{{ form.new_password2.errors }}
+                                <label for="id_new_password2">Confirm password:</label></td>
+                            <td>{{ form.new_password2 }}</td>
+                        </tr>
+                        <tr>
+                            <td></td>
+                            <td><input type="submit" value="Change my password" / class="btn"></td>
+                        </tr>
+                    </table>
```

Comment:
I think I'd probably do `{{ form.as_p }}` here too for consistency. not sure why the docs went through this manually. Also random / in the middle of your submit button